src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -112,3 +114,15 @@ func TestLoggly(t *testing.T) {
 		assert.Equal(t, 100, len(result["message"].(string)))
 	}
 }
+
+func TestIncludeInSample(t *testing.T) {
```

Comment:
@myleshorton I added a unit test for includeInSample here.  Turns out it found a bug :)  I needed to use LittleEndian instead of BigEndian.

I'll add additional unit tests in the other PR.